RomeALincei/ANLcr2.xml

```diff
@@ -28,337 +29,594 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 <msItem xml:id="ms_i1.1.2">
-                                    <locus from="14vb" to="15ra"></locus>
+                                    <locus from="14vb" to="15ra"/>
                                     <title>Introduction to the miracles of Michael</title>
+                                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ንዌጥን፡ በረድኤተ፡ አግዚአብሔር፡ ሕያው፡ ጽሒፈ፡
+                                        ተአምሪሁ፡ ለቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ትንብልናሁ፡ የሃሉ፡ <pb n="15r"/><cb n="a"/> ምስለ፡ ፍቁሩ፡ <space reason="owner"/>
+                                        ለዓለመ፡ ዓለም፡ አሜን። <gap reason="ellipsis"/></incipit>
                                 </msItem>
+                                
                                 <msItem xml:id="ms_i1.1.3">
-                                    <locus from="15va" to="16rb"></locus>
-                                    <title ref="LIT1295Dersan#MiracleHedar"></title>
+                                    <locus from="15va" to="16rb"/>
+                                    <title ref="LIT1295Dersan#MiracleHedar"/>
+                                    <incipit xml:lang="gez">ተአምሪሁ፡ ለቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ትንብልናሁ፡
+                                        የሃሉ፡ ምስለ፡ ፍቁሩ፡ <space reason="owner"/> ለዓለም፡ አሜን። እከሥት፡ በምሳሌ፡ አፉየ፡ ወእነግር፡ አምሳሌ፡ ዘእም፡
+                                        ትካት፡ ይፌክር፡ <gap reason="ellipsis"/> <cb n="b"/> <gap reason="ellipsis"/>
+                                        ወናሁ፡ እዜንወክሙ፡ በቀዳሚ፡ መዋዕል፡ በባሕር፡ ብዙኃን፡ ሰብእ፡ እምብሔረ፡
+                                        ግብጽ፡ በማዕዶተ፡ ዮርዳኖስ።</incipit>
                                 </msItem>
+                                
                                 <msItem xml:id="ms_i1.1.4">
-                                    <locus from="16rb" to="17vb"></locus>
-                                    <title ref="LIT1295Dersan#CommemorativeHedar">Apparition of St Michael to Joshua</title>
+                                    <locus from="16rb" to="17vb"/>
+                                    <title ref="LIT1295Dersan#CommemorativeHedar">Apparition of St
+                                        Michael to Joshua</title>
+                                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> አመ፡ ፲ወ፪ ለኅዳር፡ በዛቲ፡ ዕለት፡ ተዝካረ፡ በዓሉ፡
+                                        ለቅዱስ፡ ሚካኤል፡ ወርእሰ፡ ሰማያውያን፡ መሐረ፡ ለዘመደ፡ እጓለ፡ እመሕያው፡ ዘይቀውሙ፡ ኵሎ፡
+                                        ጊዜ፡ ቅድመ፡ መንበሩ፡ ለንጉሠ፡ ሰማይ፡ ወይተንብል፡ በእንቲአነ፡ ወዝ፡ ውእቱ፡ ዘርእዮ፡ ኢያሱ፡
+                                        ወልደ፡ ነዌ፡ በአምሳለ፡ ሐራሁ፡ ለንጉሥ፡ ወደንገፁ፡ እምኔሁ፡</incipit>
                                 </msItem>
+                                
                                 <msItem xml:id="ms_i1.1.5">
-                                    <locus from="17vb" to="18ra"></locus>
+                                    <locus from="17vb" to="18ra"/>
                                     <title type="complete" ref="LIT1295Dersan#SalamHedar"/>
```

Comment:
SalamCompassionate and SalamDesert as far as I can see, now they have become 2 (?)